ingestors/vaccinespotter.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+from datetime import datetime, timezone
+
+import httpx
+from smart_open import open
+
+from . import header_dict, jsonschema
+
+logger = logging.getLogger("vaccinespotter")
+
+
+def fetch(state, geojson_file):
```

Comment:
This is a fine first pass, but I expect we can make these generic, and then re-use them for different sources.

For example:
1. `fetchers/http.py`: With this method to call a url and dump the results to a file.
2. `parsers/geojson.py`: Converts a geojson file to an ndjson line per feature.
3. `normalizers/vaccinespotter.py`: Converts a vaccinespotter ndjson file into a VaccinateCA Locations schema ndjson file